src/Vegetation/PlantHydraulics.jl

```diff
@@ -0,0 +1,471 @@
+# To do: move constants out of src/ code and pass through, via structs. 
```

Comment:
could remove these comments now?

Reply:
ah darn I was just running the final tests and forgot this - will remove now

Reply:
still have to convert to bulk plant though with RAI, SAI, LAI (next PR)